DESCRIPTION

```diff
@@ -13,7 +13,7 @@ Authors@R: c(person("Paul Boutros", role = c("aut", "cre"), email = "PBoutros@me
              person("Robert Gentleman", role = "ctb"),
              person("Ross Ihaka", role = "ctb"),
              person("Caden Bugh", role = "ctb"),
-	     person("Dan Knight", role = "ctb"))
+	    	 person("Dan Knight", role = "ctb"))
```

Comment:
Was just trying to line this up with the other contributor lines. Seems that the file was using spaces instead of tabs here, so I fixed that in a separate commit.

(Same with above comment from @tyamaguchi-ucla)